bazel-base: magmad test failures

Seeing that some magmad unit tests are failing inside bazel-base
<img width="851" alt="Screen Shot 2022-03-02 at 8 18 50 PM" src="https://user-images.githubusercontent.com/37634144/156477677-d0741aa8-67e6-4c38-87bb-90eed915c4c8.png">

All the errors seem to be related:
```
orc8r/gateway/python/magma/magmad/metrics.py:22: in <module>
    from magma.common.service import MagmaService
orc8r/gateway/python/magma/common/service.py:25: in <module>
    import pkg_resources
E   ModuleNotFoundError: No module named 'pkg_resources'
```

## Repro steps
1. checkout latest master
2. cd magma/experimental/bazel-base; docker-compose build; docker-compose run magma-builder bash
3. bazel test //...

--- a/orc8r/gateway/python/magma/common/BUILD.bazel
+++ b/orc8r/gateway/python/magma/common/BUILD.bazel
@@ -64,6 +64,7 @@ py_library(
         "//orc8r/protos:metricsd_python_proto",
         "//orc8r/protos:service303_python_grpc",
         requirement("prometheus_client"),
+        requirement("setuptools"),
     ],
 )
 

